src/test/java/org/jabref/logic/database/DuplicateCheckTest.java

```diff
@@ -525,7 +524,9 @@ void compareOfTwoEntriesWithSameContentAndMixedLineEndingsReportsNoDifferences()
         assertTrue(duplicateChecker.isDuplicate(entryOne, entryTwo, BibDatabaseMode.BIBTEX));
     }
 
-    @Disabled("Book entries can have the same ISBN due to different chapters. The Test fails as crossref identifies both entries as the same.")
+    /**
+     *     Book entries can have the same ISBN due to different chapters. The Test fails as crossref identifies both entries as the same.
```

Comment:
Use one space after `*` -- normal java doc style.

Modify the text. The test DOES NOT fail!